integration/command.go

```diff
@@ -59,6 +59,7 @@ type CLICommand struct {
 }
 
 // CLIOutput represents the output from executing a CLICommand.
+// nolint:unused
```

Comment:
Why not remove this?

Reply:
This is part of a framework Mike and I created to be able to do integration tests, it was a little hard to do and run those tests consistently especially because of the way the input was managed. It didn't go far, but it's something that we can improve. I've just removed the linting error.